vsintegration/src/FSharp.Editor/CodeFixes/CodeFixHelpers.fs

```diff
@@ -106,3 +108,57 @@ module internal CodeFixExtensions =
                 let! sourceText = ctx.GetSourceTextAsync()
                 return RoslynHelpers.TextSpanToFSharpRange(ctx.Document.FilePath, ctx.Span, sourceText)
             }
+
+// This cannot be an extension on the code fix context
+// because the underlying GetFixAllProvider method doesn't take the context in.
+#nowarn "3511" // state machine not statically compilable
+
+[<AutoOpen>]
+module IFSharpCodeFixProviderExtensions =
+    type IFSharpCodeFixProvider with
+
+        member provider.RegisterFsharpFixAll() =
```

Comment:
Can you please measure allocations here, it looks like there are quite a bunch of distinct sequences and options are used.

Reply:
I didn't notice anything worrying there but maybe we can catch offline on this, not sure if I am measuring stuff correctly here